### ISSUE TITLE
*cobwebs*

### ISSUE THREAD

**Issue by jm3** [clobber/pleasedeletefacebook.com#1]
looks like [pleasedeletefacebook.com](http://pleasedeletefacebook.com) hasn't been updated in a minute… want some PRs? :)

**Comment by clobber** [comment]
Yes please, absolutely. I bookmarked but missed out on a lot of important events for 2014 and haven't gotten around to updating the site.
